std/algorithm/comparison.d

```diff
@@ -61,7 +61,7 @@ import std.range.primitives;
 import std.traits;
 // FIXME
 import std.typecons; // : tuple, Tuple, Flag;
-import std.meta : allSatisfy;
+import std.meta; // : allSatisfy, anySatisfy;
```

Comment:
If this was released, don't comment it out.  The idea is to keep things the same, not to remove the leak.